checker/result.go

```diff
@@ -91,3 +106,48 @@ func (r *Result) addCheck(checkResult *Result) {
 	// SetStatus sets Result's status to the most severe of any individual check
 	r.Status = SetStatus(r.Status, checkResult.Status)
 }
+
+// IDs for checks that can be run
+const (
+	Connectivity     = "connectivity"
+	STARTTLS         = "starttls"
+	Version          = "version"
+	Certificate      = "certificate"
+	MTASTS           = "mta-sts"
+	MTASTSText       = "mta-sts-text"
+	MTASTSPolicyFile = "mta-sts-policy-file"
+	PolicyList       = "policylist"
+)
+
+// Text descriptions of checks that can be run
+var checkNames = map[string]string{
+	Connectivity:     "Server connectivity",
+	STARTTLS:         "Support for STARTTLS",
+	Version:          "Secure version of TLS",
+	Certificate:      "Valid certificate",
+	MTASTS:           "Implementation of MTA-STS",
```

Comment:
"Inbound MTA-STS support"

Reply:
Oh woops - I forgot to say that the strings and template text are basically dummy text I added - feel free to directly push edits. If it's just the ones in this review I'm happy to add them, but if you wanna tweak more things feel free :)